streaming_providers/qbittorrent/utils.py

```
@@ -148,6 +156,15 @@ async def initialize_qbittorrent(user_data: UserData) -> APIClient:
             raise ProviderException(
                 "Invalid qBittorrent credentials", "invalid_credentials.mp4"
             )
+        except ClientResponseError as err:
+            if err.status == 403:
+                raise ProviderException(
+                    "Invalid qBittorrent credentials", "invalid_credentials.mp4"
+                )
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)